### PR TITLE
fix(conf): Make LV_COLOR_MIX_ROUND_OFS configurable 

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

And make the generate lv_config_internal.h more readable.

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: lv_conf_template.h [hunk 1]
```diff
@@ -34,6 +34,21 @@
  *Requires `LV_COLOR_DEPTH = 32` colors and the screen's `bg_opa` should be set to non LV_OPA_COVER value*/
 #define LV_COLOR_SCREEN_TRANSP 0
 
+/**
```
Comment (kisvegabor) [inline lv_conf_template.h]: I might sound odd, but from the user perspective, a so marginal config appearing at the top of the config file seems noisy.

What about this?
```c
/* Adjust color mix functions rounding. GPUs might calculate color mix (blending) differently.
 * 0: round down, 64: round up from x.75, 128: round up from half, 192: round up from x.25, 254: round up */
#if LV_COLOR_DEPTH == 32
#define LV_COLOR_MIX_ROUND_OFS (LV_COLOR_DEPTH == 32 ? 0 : 128)
#endif
```

Comment (xiaoxiang781216) [reply lv_conf_template.h]: Done.

**Comment by kisvegabor** [comment]
Thanks!

File: lv_conf_template.h [hunk 2]
```diff
@@ -34,6 +34,14 @@
  *Requires `LV_COLOR_DEPTH = 32` colors and the screen's `bg_opa` should be set to non LV_OPA_COVER value*/
 #define LV_COLOR_SCREEN_TRANSP 0
 
+/* Adjust color mix functions rounding. GPUs might calculate color mix (blending) differently.
+ * 0: round down, 64: round up from x.75, 128: round up from half, 192: round up from x.25, 254: round up */
+#if LV_COLOR_DEPTH == 32
```
Comment (kisvegabor) [inline lv_conf_template.h]: What about the one-line version?
```c
#define LV_COLOR_MIX_ROUND_OFS (LV_COLOR_DEPTH == 32 ? 0: 128)
```

Comment (xiaoxiang781216) [reply lv_conf_template.h]: Done.

**Comment by kisvegabor** [comment]
Thank you!

**Pull request merged**